examples/federated-policies/README.rst

```diff
@@ -0,0 +1,166 @@
+
+System Requirements
+-------------------
+
+1. Install Python 3.8 and NVFlare
+::
+   pip install nvflare
+
+2. The example is part of the NVFlare source code. The source code can be obtained like this,
+::
+   git clone https://github.com/NVIDIA/NVFlare.git
+
+3. TLS requires domain names. Please add following line in :code:`/etc/hosts` file,
+::
+   127.0.0.1	server1
+
+
+Usage
+-----
+
+The example is located in :code:`NVFlare/examples/federated-policies`.
+
+Setup
+_____
+
+The :code:`project.yml` file defines all the sites and users (called admin in NVFlare)
+used in the examples. The startup kits will be created by :code:`setup.sh`
+::
+    cd NVFlare/examples/federated-policies
+    ./setup.sh
+All the startup kits will be generated in this folder,
+::
+    workspace/fed_policy/prod_00
+
+.. note::
+   :code:`workspace` folder is removed everytime :code:`setup.sh` is run. Please do not save customized
+   files in this folder.
+
+Starting NVFlare
+________________
+
+This script will start up the server and 2 clients,
+::
+   ./start.sh
+
+Logging with Admin Client
+_________________________
+
+In secure mode, NVFlare creates one startup kit for each user. There are 5 users in
+this example so there are 5 folders for admin login under :code:`workspace/fed_policy/prod_00` folder.
+
+To login as an user, the appropriate folder must be selected.
+
+For example, this is how to login as :code:`admin@a.org` user,
+::
+    cd workspace/fed_policy/prod_00/trainer@a.org
+    ./startup/fl_admin.sh
+At the prompt, enter the user email :code:`admin@a.org`
+
+Multiple users can login at the same time by using multiple terminals.
+
+The setup.sh copies the jobs folder to the workspace folder. Job can be submitted like this,
+::
+   submit_job ../../jobs/job1
+
+Shutting down NVFlare
+_____________________
+All NVFlare server and clients can be stopped by using this script,
+::
+   ./stop.sh
+
+Participants
+------------
+Site
+____
+* :code:`server1`: NVFlare server
+* :code:`site_a`: Client owned by a.org with a customized authorization policy, which only allows
+users from the same org to submit job.
+* :code:`site_b`: Client owned by b.org with a customized privacy policy. The policy defines
+two scopes :code:`public` and :code:`private`. A custom filter is applied to :code:`private`.
+
+Users
+_____
+* :code:`super@a.org`: Super user with role :code:`project_admin` who can do everything
+* :code:`admin@a.org`: Admin for a.org with role :code:`org_admin`
+* :code:`trainer@a.org`: Lead trainer for a.org with role :code:`lead`
+* :code:`trainer@b.org`: Lead trainer for b.org with role :code:`lead`
+* :code:`user@b.org`: Regular user for b.org with role :code:`member`
+
+Jobs
+____
+All the jobs run the same app (numpy-sag) but have different scopes defined in :code:`meta.json`.
+
+* job1: Scope is :code:`public`
+* job2: Scope is :code:`private`
+* job3: It has no scope defined.
+* job4: It defines an non-existent scope :code:`foo`
+
```

Comment:
Can we in setup.sh create all these jobs? (like copy from numpy-sag) and generate meta.json.

So we don't need to have all these duplicate config_fed_client.json/config_fed_server.json